apps/frontend/src/pages/auth/libs/components/reset-password-form/reset-password-form.tsx

```diff
@@ -78,7 +109,13 @@ const ResetPasswordForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
 					placeholder="*******"
 					type={isConfirmPasswordVisible ? "text" : "password"}
 				/>
-				<Button label="RESET PASSWORD" type="submit" />
+				<Button
+					isDisabled={
+						!isValid || newPasswordValue.length !== confirmPasswordValue.length
```

Comment:
`newPasswordValue.length !== confirmPasswordValue.length`  was done to prevent the situation where user sends a password with a typo to the server

Reply:
The request will not be sent to the backend if the password and confirmPassword are not the same. This should be checked in handleFormSubmit (I believe it should already be there)

Reply:
in requirements for resetPassword, we have this:

Save Password button remains disabled until all mandatory fields are filled correctly.

Our validation schema doesn't have a field for confirmPassword because it is also used on backend, where confirm password value shouldn't be present. If we don't have this check in this form it is possible to press reset password button when confirmPassword field is empty, which is bad from perspective of UI in my opinion